### PR TITLE
remote: adds support for s3.list_objects

### PR DESCRIPTION
**Pull request by bwalsh**
Adds documentation for s3.list_objects.   See https://github.com/iterative/dvc/pull/1708

### CODE REVIEW

File: static/docs/commands-reference/remote:modify.md [hunk 1]
```diff
@@ -89,6 +89,13 @@ these settings, you could use the following options:
     $ dvc remote modify myremote use_ssl false
   ```
 
+* `listobjects` - whether or not to use `list_objects`. By default, `list_objects_v2` is used. Useful for ceph and other s3 emulators.
```
Comment (shcheklein) [inline static/docs/commands-reference/remote:modify.md]: please, keep the width under 80 symbols

File: static/docs/commands-reference/remote:modify.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ these settings, you could use the following options:
     $ dvc remote modify myremote use_ssl false
   ```
 
-* `listobjects` - whether or not to use `list_objects`. By default, `list_objects_v2` is used. Useful for ceph and other s3 emulators.
```
Comment (shcheklein) [inline static/docs/commands-reference/remote:modify.md]: Sorry, I think there is some misunderstanding. I didn't mean to keep the comment length under 80. Actually the initial text you put together is great! And it's good to know (being able to search for ceph, for example)! I just wanted to wrap the lines at 80. Multiple lines are absolutely fine. Thank you for PR!

Comment (bwalsh) [reply static/docs/commands-reference/remote:modify.md]: @shcheklein  should be fixed now.  Let me know if other issues.

**Comment by shcheklein** [comment]
@bwalsh  Sorry, I think there is some misunderstanding. I didn't mean to keep the comment length under 80. Actually the initial text you put together is great! And it's good to know (being able to search for ceph, for example)! I just wanted to wrap the lines at 80. Multiple lines are absolutely fine, could you return the previous version you had and just wrap it properly? :) Thank you for the PR!

**Pull request merged**